[nnc] enable tensorexpr quantize/dequantize

IvanKobzarev opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #70193
* #70189
* #70188

Differential Revision: [D33239618](https://our.internmc.facebook.com/intern/diff/D33239618)

IvanKobzarev committed:
[nnc] enable tensorexpr quantize/dequantize

[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33239618).

IvanKobzarev committed:
Update on "[nnc] enable tensorexpr quantize/dequantize"


Differential Revision: [D33239618](https://our.internmc.facebook.com/intern/diff/D33239618)

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Do we have any perf measurements for this?

IvanKobzarev committed:
Update on "[nnc] enable tensorexpr quantize/dequantize"


Differential Revision: [D33239618](https://our.internmc.facebook.com/intern/diff/D33239618)

[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33239618).
